app/templates/services/previous_services.html

```diff
@@ -1,6 +1,4 @@
 {% extends "_base_page.html" %}
-{% import "toolkit/summary-table.html" as summary %}
-{% import "macros/submission.html" as submission %}
 
 {% block pageTitle %}
   Copy your {{ framework.name }} services – Digital Marketplace
```

Comment:
Is this an accurate title? It should probably be "Copy your {{ source_framework.name }} services"

Reply:
good spot 👀   fixed in 1769ba7